src/m365/entra/commands/pim/pim-role-assignment-eligibility-list.ts

```diff
@@ -74,6 +76,9 @@ class EntraPimRoleAssignmentEligibilityListCommand extends GraphCommand {
       },
       {
         option: '--includePrincipalDetails [includePrincipalDetails]'
+      },
+      {
+        option: '--withPrincipalDetails [withPrincipalDetails]'
       }
```

Comment:
since it's a flag we should specify it this way

```suggestion
      {
        option: '--includePrincipalDetails'
      },
      {
        option: '--withPrincipalDetails'
      }
```